rhino/src/main/java/org/mozilla/javascript/Node.java

```diff
@@ -1165,11 +1165,16 @@ private void toString(ObjToIntMap printIds, StringBuilder sb) {
                         }
                         break;
                     case OBJECT_IDS_PROP:
+                    case OBJECT_IDS_COMPUTED_PROP:
                         {
                             Object[] a = (Object[]) x.objectValue;
                             sb.append("[");
                             for (int i = 0; i < a.length; i++) {
-                                sb.append(a[i].toString());
+                                if (a[i] != null) {
```

Comment:
I set Token.printTrees to true for debugging and got `NullPointerException`.
https://github.com/mozilla/rhino/blob/cc302b4f8fac284145c51f5ceea21ec333f7a1c8/rhino/src/main/java/org/mozilla/javascript/Token.java#L27

This PR now displays the following:
```
OBJECTLIT [object_ids_computed_prop: [COMPUTED_PROPERTY, COMPUTED_PROPERTY, <empty>]] [object_ids_prop: [<empty>, <empty>, x]]
```